src/TextInput/index.js

```diff
@@ -85,60 +114,70 @@ class TextInput extends React.Component<Props, State> {
       prefix,
       onChange,
       inlineLabel,
+      suffix,
     } = this.props;
     const { focused } = this.state;
     return (
-      <View>
-        {label && !inlineLabel && (
-          <FormLabel filled={!!value} required={required} disabled={disabled}>
-            {label}
-          </FormLabel>
-        )}
-        <View
-          style={[
-            styles.inputContainer,
-            size === 'normal' ? styles.normalSize : styles.smallSize,
-            disabled
-              ? styles.inputContainerDisabled
-              : styles.inputContainerDefault,
-            focused
-              ? styles.inputContainerBorderFocused
-              : styles.inputContainerBorderDefault,
-          ]}
-        >
-          {prefix && <Prefix size={size}>{prefix}</Prefix>}
-          {label && inlineLabel && (
-            <InlineLabel>
-              <FormLabel
-                filled={!!value}
-                inlineLabel
-                required={required}
-                disabled={disabled}
-              >
-                {label}
-              </FormLabel>
-            </InlineLabel>
+      <TouchableWithoutFeedback onPress={this.focusTextInput}>
+        <View style={styles.inputWrapper}>
+          {label && !inlineLabel && (
```

Comment:
this will crash on android if label = empty string

Reply:
Ok, will check it.